SADL/AM-CDM-material.sadl

```diff
@@ -30,6 +30,11 @@ Feedstock (note "Information about a feedstock used as input to an additive manu
 	described by materialProperties (note "Measured properties of the material.") with values of type MaterialPropertiesType,
 	described by mixedFeedstock (note "Parent or parents of feedstock, if it is blended or mixed.") with values of type ParentFeedstock.
 
+ContinuousMaterial is a class,

```

Comment:
provide a note for what you mean by this

Reply:
To be considered in pull request #17